rel-eng/atomic-enterprise-master.service

```diff
@@ -3,16 +3,14 @@ Description=Atomic Enterprise Master
 Documentation=https://github.com/openshift/origin
 After=network.target
 After=etcd.service
-Before=atomic-enterpries-node.service
-Before=atomic-enterprise-sdn-node.service
-Before=atomic-enterprise-sdn-master.service
+Before=atomic-enterprise-node.service
 Requires=network.target
 
 [Service]
 Type=notify
-EnvironmentFile=/etc/sysconfig/atomic-enterprise-master
+EnvironmentFile=/etc/sysconfig/origin-master
 Environment=GOTRACEBACK=crash
-ExecStart=/usr/bin/atomic-enterprise start master --config=${CONFIG_FILE} $OPTIONS
+ExecStart=/usr/bin/openshift start master --config=${CONFIG_FILE} $OPTIONS
```

Comment:
I'm fine with the unit files referencing `atomic-enterprise` for clarity.  I think we should at least ship all the same symlinks as upstream though.  I do think they recently added an `origin` symlink.  We want to make sure someone running a command or write a tool for upstream doesn't have to translate it into atomic-enterprise.